lib/ddtrace/contrib/sucker_punch/instrumentation.rb

```diff
@@ -43,6 +45,10 @@ def perform_in(interval, *args)
 
             private
 
+            def datadog_configuration
```

Comment:
why `datadog_configuration` here instead of `configuration`?

Reply:
Because this particular patch rewrites methods on the target class, instead of injecting a module or using some separate tracer object. There's a decent chance for method name collision with `configuration` for an object called `Job`, hence we prefixed with `datadog` here. Just a pro-active measure to prevent conflicts.

It would be nice to make these integration implementations a bit more consistent at some point, but that'll have to be done in future refactors.

Reply:
:+1: makes sense, I sort of assumed this was the case.